chromadb/db/clickhouse.py

```diff
@@ -57,7 +57,7 @@ def _init_conn(self):
         self._conn = clickhouse_connect.get_client(
             host=self._settings.clickhouse_host, port=int(self._settings.clickhouse_port)
         )
-        self._conn.query(f"""SET allow_experimental_lightweight_delete = 1;""")
+        self._conn.query(f"""SET allow_experimental_lightweight_delete = true;""")
```

Comment:
For my understanding - does this actually make a difference? AFAIK this is the same as = 1

Reply:
it seems to 🤷

Reply:
can confirm from my end that I hit the error as defined in https://github.com/ClickHouse/ClickHouse/pull/37893/files#diff-9b3ae52f39d9c550bfce3a8babdbc528e8f5c00c38c5a88ccca6263705f7d19cR40 when `true` is used instead of `1`.

We can also verify with
https://github.com/ClickHouse/dbt-clickhouse/blob/4dabe1459323c80fb633095c9b04ff7eefae94fd/dbt/adapters/clickhouse/dbclient.py#L124

Reply:
We tested this - both work actually! Going to go back to `1` because there is "no reason" to change it.